app/repositories/leaderboards.py

```diff
@@ -185,8 +184,6 @@ async def fetch_user_score(
                 row_number() OVER (PARTITION BY s.userid ORDER BY s.{sort_column} DESC) score_order_rank
             FROM {scores_table} s
             INNER JOIN users ON users.id = s.userid
-            LEFT JOIN clans c ON users.clan_id = c.id
-            INNER JOIN users_stats ON users_stats.id = s.userid
```

Comment:
these are in other queries too - should they also be removed?

Reply:
removed where possible